packages/connect/CHANGELOG.md

```diff
@@ -10,41 +10,37 @@
 
 Use the persistent link [connect.trezor.io/9](https://connect.trezor.io/9/) to access the latest stable version of Connect Explorer.
 
-# 9.3.1-beta.3
+# 9.4.0
 
 ## connect
 
+We introduce a new init option `coreMode`, which allows websites to interact either using the existing `iframe` mode or the new `popup` mode. The `popup` mode is allows to use WebUSB even on third-party websites, which is not possible with the `iframe` mode. By default, the `coreMode` is set to `auto`, which means that the library will automatically choose the best mode based on the environment. More information can be found in [init method documentation](https://connect.trezor.io/9.3.1-beta.1/methods/other/init/).
```

Comment:
mode **is** allows to use

Reply:
I modified all that section for better readability and fixed that mistake. https://github.com/trezor/trezor-suite/pull/13923/commits/4f5da110f67bb9bbe93b415038d0d36bb6ad0245